moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -3,51 +3,26 @@ import { gql } from "apollo-boost";
 
 export const useGetProjectListView = ({
   columnsToReturn,
-  exportColumnsToReturn,
   queryLimit,
   queryOffset,
   orderByColumn,
   orderByDirection,
   searchWhereString,
   advancedSearchWhereString,
+  queryName,
 }) => {
-  const { query, exportQuery } = useMemo(() => {
-    const queryString = `query ProjectListView {
-        project_list_view (
-            limit: ${queryLimit}
-            offset: ${queryOffset}
-            order_by: {${orderByColumn}: ${orderByDirection}}
-            where: { 
-              ${advancedSearchWhereString ? advancedSearchWhereString : ""}
-              ${searchWhereString ? `_or: [${searchWhereString}]` : ""} 
-            }
-        ) {
-            ${columnsToReturn.join("\n")}
-        },
-        project_list_view_aggregate (
-          where: { 
-            ${advancedSearchWhereString ? advancedSearchWhereString : ""}
-            ${searchWhereString ? `_or: [${searchWhereString}]` : ""} 
-          }
-        ) {
-          aggregate {
-            count
-          }
-        }
-      }`;
-    const query = gql`
-      ${queryString}
-    `;
-
-    const exportQueryString = gql`{
+  const { query } = useMemo(() => {
+    const queryString = `query ${queryName} {
```

Comment:
`queryName` is rly helpful for debugging which query is sending network requests, the export or the list view